db_test.go

```diff
@@ -8,13 +8,15 @@ import (
 
 	"github.com/stretchr/testify/suite"
 	"github.com/surrealdb/surrealdb.go"
+	"github.com/surrealdb/surrealdb.go/pkg/gorilla"
+	"github.com/surrealdb/surrealdb.go/pkg/iwebsocket"
 )
 
 // TestDBSuite is a test s for the DB struct
 type SurrealDBTestSuite struct {
 	suite.Suite
-	db      *surrealdb.DB
-	options []surrealdb.Option
+	db *surrealdb.DB
+	ws iwebsocket.IWebSocket
```

Comment:
We may want to have this tied to a specific implementation OR tied to general properties of a websocket. I think we do want the properties, but then this may be a map 
```
wsImplementations map[string]websocket.WebSocket{
   "gorilla": gorilla.Create(),
}
```
so that in the tests we can have
```
func TestConnects(t *testing.T) {
   for wsName, ws := range suite.wsImplementations {
      suite.Run(fmt.Sprintf("_%+v", wsName), func() {
         <TEST_CODE_HERE>
      })
   }
}

Reply:
done